package.json

```diff
@@ -24,5 +24,8 @@
     "imagemin-mozjpeg": "^8.0.0",
     "imagemin-pngquant": "^8.0.0",
     "imagemin-webp": "^5.1.0"
+  },
+  "dependencies": {
+    "http-server": "^0.12.3"
```

Comment:
We should be careful not to add `http-server` as a project dependency. That means that it is required by the project in production. 

In general, I don't think we want to add it as a dev dependency either. It is just a tool that you need on your machine. 

You need to remove lines 28-30. Then run `npm install` so it removes the dependency from `package-lock.json`. Commit the changes to `package-lock.json`. 

You probably need to install the server globally `npm install http-sever -g`. That makes it global on your machine.